iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -366,6 +366,20 @@ TEST_F(PoshRuntime_test, getMiddlewarePublisherPublisherlistOverflow)
     EXPECT_TRUE(publisherlistOverflowDetected);
 }
 
+#if defined(RESTRICT_TO_1_TO_N_COMMUNICATION)
```

Comment:
Please do not use macros here. Is it possible to use the roudi environment and create a port manager which supports and one which does not support n to m communication?

Reply:
Yeah, I hate macros, too 😃 In #281 we decided to go forward with the macro approach and replace it later on. I'm happy to do that as part of this pull request.

Reply:
@mossmaurice I fixed the macro approach in the pull request: https://github.com/eclipse/iceoryx/pull/295

Reply:
I already fixed it on my local branch in a nicer SFINAE way ;-) Could you revert your changes?

Reply:
@mossmaurice No problem, I will do that on monday!

Reply:
Removed the macros